packages/angular/cli/src/utilities/package-metadata.ts

```diff
@@ -249,6 +249,11 @@ export async function fetchPackageMetadata(
     ...(registry ? { registry } : {}),
   });
 
+  if (!response.versions) {
+    // While pacote type stats that versions cannot be undefined this is not the case.
```

Comment:
nit: ```suggestion
    // While pacote type states that versions cannot be undefined this is not the case.
```
Same for the other change.